pkg/controller/.import-restrictions

```diff
@@ -244,7 +244,7 @@
         "k8s.io/kubernetes/pkg/util/goroutinemap/exponentialbackoff",
         "k8s.io/kubernetes/pkg/util/hash",
         "k8s.io/kubernetes/pkg/util/labels",
-        "k8s.io/kubernetes/pkg/util/metrics",
+        "k8s.io/component-base/metrics/prometheus/ratelimiter",
```

Comment:
same comment about dropping this line from a `"SelectorRegexp": "k8s[.]io/kubernetes/pkg",` section